src/Elements/Spatial/Grid2d.cs

```diff
@@ -100,8 +100,8 @@ public Grid2d(double uDimension, double vDimension)
         /// The polygon's bounding box parallel to the supplied transform will be
         /// used as the grid extents. 
         /// </summary>
-        /// <param name="boundary"></param>
-        /// <param name="t">A transform representing the </param>
+        /// <param name="boundary">The external boundary of this grid system.</param>
+        /// <param name="t">A transform representing the alignment of the grid.</param>
```

Comment:
Can we rename `t` to `transform`? Apologies that I haven't caught this in other places. I'm trying to make this library as friendly to devs getting started as possible, so I'm erring on the side of legibility.

Reply:
Done.